src/nvim/api/window.c

```diff
@@ -19,6 +19,45 @@
 #include "nvim/window.h"
 #include "nvim/screen.h"
 #include "nvim/move.h"
+#include "nvim/fold.h"
+
+/// Create a fold from line "start" to line "end" (inclusive) in the current
+/// window.
+/// @param window
+/// @param start starting line
+/// @param end starting end
+/// @param err
+void nvim_win_add_fold(Window window, Integer start, Integer end, Error *err)
```

Comment:
Does it overwrite a fold set with the same arguments? If so, it should be named `nvim_win_set_fold` rather than "add".

Reply:
good remark. `zf2j` twice creates 2 identical folds, and they have to be deleted separately so it seems like add_fold is the correct choice. Now I wonder if beiing able to create 2 identical folds makes sense :s